lib/crack.rb

```diff
@@ -1,18 +1,41 @@
 require_relative './enigma'
+require_relative './valid_input'
 
 ARGV == ['ecnrypted.txt', 'cracked.txt', 'date']
 ARGV[0] == 'encrypted.txt'
 ARGV[1] == 'cracked.txt'
 ARGV[2] == 'date'
 
-encrypted = File.open(ARGV[0], 'r')
-message = encrypted.read
-
+valid_input = ValidInput.new(ARGV[0], ARGV[1], ARGV[2])
```

Comment:
initialization of the valid_input object with the arguments passed in from the command line